### ISSUE TITLE
Add local chart testing

### ISSUE THREAD

**Issue by darren-the** [darren-the/crypto-trading-v2#12]
Enables outputting the results of a local pipeline straight to a chart

**Comment by darren-the** [comment]
Has been addressed in PR #29. Does not yet allow for viewing results live as the pipeline is running, only once the pipeline is finished. Live viewing of a chart will be implemented with a new issue

**Comment by darren-the** [comment]
Live chart viewing #31
